doc/security/specs/identities_and_root_keys/README.md

```diff
@@ -253,39 +243,118 @@ OwnerIntermediateKey =
-    Owner software. The ROM Extension implements this property.
+For the sealing chain, this register uses a diversification value determined by
+ROM_EXT.
+    </td>
+  </tr>
+</table>
+
+## OwnerIntermediateIdentity
+
+The OwnerIntermediateIdentity is associated with CDI_0 in the DICE attestation
+chain.
+
+```
+OwnerIntermediateIdentitySeed =
+  KM_DERIVE(OwnerIntermediateKey, OTBNDiversificationConstant)
+
+// ASYM_KDF is a KDF function compliant to the Asymmetric Key
+// requirements defined in the Attestation specification document.
+OwnerIntermediateIdentity_Private = ASYM_KDF(OwnerIntermediateIdentitySeed)
+CLEAR_BEFORE_NEXT_BOOT_STAGE(OwnerIntermediateIdentitySeed, OwnerIntermediateIdentity_Private)
+```
+
+<table>
+  <tr>
+    <td><strong>Name</strong></td>
+    <td><strong>Encoding</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td>OwnerIntermediateIdentitySeed</td>
+    <td>SW Register Output</td>
+    <td>Seed used to generate the OwnerIntermediateIdentity asymmetric key.</td>
+  </tr>
+  <tr>
+    <td>OTBNDiversificationConstant</td>
+    <td>Gates</td>
+    <td>
+A constant defined in gates. Used to diversify sideloaded values to OTBN.
+
+Hidden from software.
+    </td>
+  </tr>
+</table>
+
+The `OwnerIntermediateIdentitySeed` and the private portion of the
+OwnerIntermediateIdentity shall be cleared before the bootloader (BL0) hands
+over execution to the kernel.
+
+## OwnerKey
+
+The `OwnerKey` is the key diversified with `OwnerSeed` and it is associated
+with the CDI_1 certificate.
+
+The `OwnerIntermediateKey` is generated as follows:
+
+```
+OwnerKey =
+   KM_DERIVE(OwnerIntermediateKey, OwnerSeed | SoftwareBindingValue)
+```
+
+<table>
+  <tr>
+    <td><strong>Name</strong></td>
+    <td><strong>Encoding</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td> id="creator-secret"OwnerSeed</td>
+    <td>Flash</td>
+    <td>
+Additional diversification key stored in flash. Provisioned during ownership
+transfer.
+
+Hidden from software once provisioned.
     </td>
   </tr>
   <tr>
     <td>SoftwareBindingValue</td>
     <td>SW Register<br>Lockable Input</td>
     <td>
-Software binding value configured during secure boot. See
-<a href="#software-binding">Software Binding</a> for more details.
+For the attestation chain, this register is populated with owner-specific
+measurements.
+
+For the sealing chain, this register uses a diversification value determined by
+owner.
     </td>
   </tr>
 </table>
 
-## Owner Identity
+## OwnerIdentity
 
-The Owner Identity is used as a cryptographic identifier bound to the device and
-the Silicon Owner. It is used in Attestation flows. The Owner Identity is not
+The OwnerIdentity is used as a cryptographic identifier bound to the device and
+the SiliconOwner. It is used in Attestation flows. OwnerIdentity is not
 expected to change during the lifetime of the device ownership assignment.
```

Comment:
that is not true. It will change if owner firmware is updated, even if the owner stays the same

Reply:
Removed the last incorrect sentence.